x-pack/plugins/infra/public/alerting/metric_threshold/components/alert_details_app_section.tsx

```diff
@@ -53,6 +60,12 @@ export function AlertDetailsAppSection({ alert, rule }: AppSectionProps) {
       dateFormat={uiSettings.get('dateFormat') || DEFAULT_DATE_FORMAT}
       id={ALERT_START_ANNOTATION_ID}
     />,
+    <AlertTimeRangeAnnotation
```

Comment:
nit: 
I would name it with a mention of the _active_ alert status. e.g. `AlertActiveAreaAnnotations`

Reply:
I didn't use `Area` in the name so that if we change the annotation from `rect annotation` to something else, we don't need to change the name.

I liked the `active` in the name of the component, so I rename it from `AlertTimeRangeAnnotation` to `AlertActiveTimeRangeAnnotation`